### PR TITLE
Switch to SLES12SP3 which is the base for SOC8.

### PR DESCRIPTION
**Pull request by jsuchome**
This should go in after we create 4.0 branch for SOC7

### CODE REVIEW

**Comment by jsuchome** [comment]
There's a conflict in `grub2` package versions.

File: scripts/install-chef-suse.sh [hunk 1]
```diff
@@ -519,8 +519,8 @@ else
 fi
 
 for arch in $product_arches; do
-    # Checks for SLE12 SP2 medias
-    MEDIA=/srv/tftpboot/suse-12.2/$arch/install
+    # Checks for SLE12 SP3 medias
```
Comment (aplanas) [inline scripts/install-chef-suse.sh]: nit: media?

File: scripts/install-chef-suse.sh [hunk 2]
```diff
@@ -534,25 +534,25 @@ for arch in $product_arches; do
         echo "Detected SUSE OpenStack Cloud Deps media."
     else
 
-        sp2mediasum="a1f512c3244c8e75da0f47d2d0752064"
+        sp3mediasum="59280f20e0bee15615b598a3ea3f027e"
```
Comment (dirkmueller) [inline scripts/install-chef-suse.sh]: are we sure that those media sums don't change? I would assume we have to comment out the check for now while SP3 isn't released yet..

Comment (jsuchome) [inline scripts/install-chef-suse.sh]: Yep, this is the sum of current state of medias. I'll comment it.

**Pull request merged**